File: MutableLanguageChange/MutableLanguageChange/ViewController.m
//
//  ViewController.m
//  MutableLanguageChange
//
//  Created by 刘春奇 on 2017/2/28.
//  Copyright © 2017年 刘春奇. All rights reserved.
//

#import "ViewController.h"
#import "HCLanguageTool.h"
@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIButton *btn;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // Do any additional setup after loading the view, typically from a nib.
    self.navigationItem.title = HCGetStringWithKeyFromTable(@"root", @"Main");
    self.navigationController.tabBarItem.title = HCGetStringWithKeyFromTable(@"home", @"Main");
    
}
- (IBAction)changeLanguage:(id)sender {
    [[HCLanguageTool shardInstance] changeNowLanguage];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 混编demo/demo/ViewController.swift
//
//  ViewController.swift
//  demo
//
//  Created by 刘春奇 on 2016/10/19.
//  Copyright © 2016年 刘春奇. All rights reserved.
//

import UIKit
import CABasicProximityKit
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        CABasicProximityManager .shared().addMonitoredBeacon("dsa")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

